source/models/groups.py

```
@@ -0,0 +1,135 @@
+                            f"Starting deletion of {len(not_working_list)} wallpapers from 'not working' group")
+        
+        deleted_count = 0
+        for wallpaper_id in not_working_list:
+            wallpaper_path = path.join(wallpaper_dir, str(wallpaper_id))
+            
+            if path.exists(wallpaper_path) and path.isdir(wallpaper_path):
+                try:
+                    rmtree(wallpaper_path)
+                    deleted_count += 1
+                    self.logger.component("GROUPS", f"Deleted wallpaper: {wallpaper_id}")
+                except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)